structured_data/2021_06_30_tensorflow_data_validation/README.md

```diff
@@ -0,0 +1,19 @@
+# TensorFlow Data Validation
+TensorFlow Data Validation (TFDV) is a tool to allow that allows you to
+detect discrepancies between datasets. Its functionality includes:
+* Visualizing your data
+* Automatically inferring a schema for your data
+* Detecting when a dataset doesn't match the schema
+* Detecting when different datasets diverge too much
+
+This chapter tip goes into the standalone version of TFDV.
+In the accompanying chapter talk, we go deeper into
```

Comment:
Can we replace this reference to the chapter talk with some URLs that *are* publicly accessible?
For non-ML6 people who are interested in more details about drift & skew metrics!

Reply:
Oops, seems I looked over this reference, will adapt!